test/e2e/performance/index.html

```diff
@@ -0,0 +1,157 @@
+        categoryAxis: {
+          position: 'top',
+          startOnAxis: true,
+          labelsEnabled: false,
+          title: '',
+          titleBold: false,
+        },
+        trendLines: [],
+        graphs: [
+          {
+            balloonText: 'TaskDuration [[value]]',
+            bullet: 'triangleDown',
+            id: 'TaskDuration',
+            lineThickness: 3,
+            precision: 3,
+            title: 'TaskDuration',
+            valueAxis: 'Duration',
+            valueField: 'TaskDuration',
+          },
+          {
+            balloonText: 'ScriptDuration [[value]]',
+            bullet: 'triangleUp',
+            id: 'ScriptDuration',
+            lineThickness: 2,
+            precision: 3,
+            title: 'ScriptDuration',
+            valueAxis: 'Duration',
+            valueField: 'ScriptDuration',
+          },
+          {
+            balloonText: 'RecalcStyleDuration [[value]]',
+            bullet: 'diamond',
+            id: 'RecalcStyleDuration',
+            lineThickness: 2,
+            precision: 3,
+            title: 'RecalcStyleDuration',
+            valueAxis: 'Duration',
+            valueField: 'RecalcStyleDuration',
+          },
+          {
+            balloonText: 'LayoutDuration [[value]]',
+            bullet: 'square',
+            id: 'LayoutDuration',
+            lineThickness: 2,
+            precision: 3,
+            title: 'LayoutDuration',
+            valueAxis: 'Duration',
+            valueField: 'LayoutDuration',
+          },
+        ],
+        guides: [],
+        valueAxes: [
+          {
+            id: 'Duration',
+            markPeriodChange: false,
+            title: 'Duration [s]',
+          },
+        ],
+        allLabels: [],
+        balloon: {},
+        legend: {
+          enabled: true,
+          spacing: 8,
+          useGraphSettings: true,
+        },
+        titles: [
+          {
+            id: 'Title-1',
+            size: 15,
+            text: title,
+          },
+        ],
+        dataProvider: data,
+      });
+    }
+  </script>
+
+  <script type="text/javascript">
+    // OS detection inspired by: https://stackoverflow.com/questions/11219582/how-to-detect-my-browser-version-and-operating-system-using-javascript
+    let os ='Unknown OS';
+    const WINDOWS ='Windows', MAC ='MacOS', UNIX ='UNIX', LINUX ='Linux';
+    if (navigator.appVersion.indexOf('Win')!==-1) os = WINDOWS;
+    if (navigator.appVersion.indexOf('Mac')!==-1) os = MAC;
+    if (navigator.appVersion.indexOf('X11')!==-1) os = UNIX;
+    if (navigator.appVersion.indexOf('Linux')!==-1) os = LINUX;
+
+    // set data file path per OS
+    let dataFilePath = '';
+    if (os === WINDOWS) {
+      dataFilePath = 'data/windows/data.js';
+    } else if (os === MAC) {
+      dataFilePath = 'data/macos/data.js';
+    } else if (os === UNIX || os === LINUX) {
+      dataFilePath = 'data/linux/data.js';
+    } else {
+      throw new Error('OS unknown');
+    }
```

Comment:
in the future, we may want to be able to see results for other os, this is ok for now.

Reply:
yes indeeed it is for the 1st phase only,
we will implement a dropdown to be able to see perf for other tests